src/services/models/gitHubModels.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { reshape } from 'utilities';
+
+export const toGitHubModel = (response = {}) =>
+  reshape(response[0], {
```

Comment:
This doesn't make sense. Is response an array? If it is the default shouldn't be an object.